dietpi/misc/dietpi-nordvpn

```diff
@@ -242,6 +242,7 @@ _EOF_
 			(( $NORDVPN_CONNECTED )) && G_WHIP_MENU_ARRAY+=('Disconnect' '')
 
 		fi
+		G_WHIP_MENU_ARRAY+=('Update' ': This will refresh NordVPN server list')
```

Comment:
Maybe: "This updates the NordVPN server list"?
It looks a bid funny when "Refresh" "Update"s something while "Update" "refresh"es something 😄.

Then maybe above as well:
```
'Refresh' ': Refresh VPN connection status'
```

Reply:
yeah I know. I was struggling with the wording quite a lot. Maybe we would need to swap. `Update` will update VPN status while `Refresh` will refresh server list

Reply:
Also "update" sounds more like a real change compared to "refresh"?

Reply:
Well you could name it `Status Update` and `Server Liste Refresh`

Looks like `chicken` or `egg` question 🤣